kurtosis-devnet/pkg/kurtosis/api/run/kurtosis_run.go

```diff
@@ -127,5 +127,21 @@ func (r *KurtosisRunner) Run(ctx context.Context, packageName string, args io.Re
 	}
 
 	return nil
+}
+
+func (r *KurtosisRunner) RunScript(ctx context.Context, script string) error {
+	if r.dryRun {
+		fmt.Printf("Dry run mode enabled, would run script %s in enclave %s\n",
```

Comment:
Tiny thing: I would add `\n` around `script` because this message will be unreadable

Reply:
yup, that's fair